src/odemis/gui/cont/tabs.py

```diff
@@ -2000,6 +2007,83 @@ def __init__(self, name, button, panel, main_frame, main_data):
         self.tb.add_tool(TOOL_ACT_ZOOM_FIT, self.view_controller.fitViewToContent)
         # TODO: add tool to zoom out (ie, all the scintillators, calling canvas.zoom_out())
 
+    def _on_btn_optical_autofocus(self, _):
+        """
+        Start or cancel the calibration when the button is triggered.
+        """
+        # from odemis.gui.comp.buttons import ImageTextButton
+        # a = ImageTextButton()
+        # a.SetLabelText
+        label_text = self.btn_optical_autofocus.GetLabelText()
+        if label_text == "Run...":
```

Comment:
Will name it to "Run". Since there might be a need to run the calibrations again, the workaround for the "status" is to change the text label for the calibration and indicate it there